rs/tests/ckbtc/ckbtc_minter_basics_test.rs

```diff
@@ -139,3 +226,12 @@ async fn test_update_balance(agent: &CkBtcMinterAgent) {
         .expect("Error while decoding response.");
     assert!(res.is_err());
 }
+
+fn main() -> Result<()> {
+    SystemTestGroup::new()
+        .with_setup(config)
```

Comment:
The more modern name for these setup functions is `setup` instead of the old `config`. Consider renaming this one and the ones below.